<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chairs</class>
 <widget class="QWidget" name="chairs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f8f8f8;</string>
  </property>
  <widget class="QWidget" name="Body" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f8f8f8;</string>
   </property>
   <widget class="QWidget" name="Header_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1051</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#555555;
color:#000000;</string>
    </property>
    <widget class="QLabel" name="lblTitle_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFFFFF;
font-size:20pt;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>&lt;User Logged in&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHome_3">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>0</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFFFFF;
font-size: 21px;
border:0;</string>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTitle_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFFFFF;
text-align: center;
font-size:26pt;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Seat Selection&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>40</y>
       <width>281</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFFFFF;
font-size: 21px;</string>
     </property>
     <property name="text">
      <string>Connection Status</string>
     </property>
    </widget>
    <widget class="QLabel" name="clock_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFFFFF;
font-size: 21px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>611</width>
      <height>501</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>611</width>
      <height>501</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QHeaderView::section { background-color:#555555; color:#FFFFFF; font-weight:bold;}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
